formulas/powers.py

```diff
@@ -115,6 +119,22 @@
     Assumptions(And(Element(a, RR), Element(b, RR), Element(c, RR), Element(d, RR), Unequal(a+b*ConstI, 0))))
 
 
+make_entry(ID("2090c3"),
+    Formula(Equal(
+        Pow(Mul(x, y), a),
```

Comment:
Can I use `(x*y)**a` instead of `Pow(Mul(x, y), a)`?

Reply:
Yes! It will still show as ``Pow(Mul(x, y), a)`` on the website, but for the Python source saving some space is fine.

Reply:
Ok: https://github.com/fredrik-johansson/fungrim/pull/12